### ISSUE TITLE
Packages for Ubuntu 16.04 LTS missing

### ISSUE THREAD

**Issue by heinlein-support** [rear/rear#939]
There are no packages for Ubuntu 16.04 at Open SuSE Build Service. Looks like somebody hast to add the new build target there. It would be helpful to have Ubuntu packages this week, since I have to finish a book this Friday where ReaR is described.

**Comment by jsmeix** [comment]
I do neither use Ubuntu nor do I know how to make
packages for Ubuntu so that I cannot help here.

But it seems you use Ubuntu and therefore I ask you
to contribute what is missing for your purpose.

**Comment by heinlein-support** [comment]
I asked Schlomo to fix it OR to give me access to the rear project at Open SuSE Build Service. I would love to fix it in just 10 seconds. Just add pheinlein to the OBS project as maintainer. Schlomo asked me, to open a ticket here instead.

What is missing? A simple click at Ubuntu 16.x repo in the settings of the Rear project at OBS.

**Comment by schlomo** [comment]
+1 for maintainer access to pheinlein on OBS. They already contributed
major code several years back.

Am 25.07.2016 9:20 vorm. schrieb "heinlein-support" <
notifications@github.com>:

**Comment by jsmeix** [comment]
@heinlein-support 
hooray!

With great pleasure I added you right now to
https://build.opensuse.org/project/users/Archiving:Backup:Rear

I also blindly added the repository named "Ubunto 16.04" at
https://build.opensuse.org/project/repositories/Archiving:Backup:Rear
but because I am not a Ubuntu user I cannot fix things here
if it fails to build for that repository or if it builds but
the result does not run well on Ubunto 16.04.

**Comment by jsmeix** [comment]
According to
https://build.opensuse.org/project/monitor/Archiving:Backup:Rear
all rear* packages built successfully for "xUbuntu_16.04".

Therefore I consider this issue to be fixed.

**Comment by jsmeix** [comment]
I also added 'pheinlein' as maintainer to
https://build.opensuse.org/project/users/Archiving:Backup:Rear:Snapshot

I also added the Ubunto 16.04 repository to
https://build.opensuse.org/project/repositories/Archiving:Backup:Rear:Snapshot

FYI:
Currently rear in Archiving:Backup:Rear:Snapshot
fails to build for all repositories but that is a different issue.

**Comment by gdha** [comment]
@heinlein-support @jsmeix has the fix been imported into the master tree of rear as well? And, what was required to fix the build? Just as a curiosity... btw thanks for the fix 👍

**Comment by jsmeix** [comment]
@gdha 
as far as I understand it this issue here is only about
to enable building rear also for Ubuntu 16.04
which means this issue here is only about to
activate Ubuntu 16.04 in the openSUSE build service.

Is your question perhaps meant for https://github.com/rear/rear/issues/940
which is about to fix something inside rear
cf. https://github.com/rear/rear/issues/940#issuecomment-234886660

**Comment by gdha** [comment]
Indeed I guess so as I had no clue what was done to fix it :)

**Comment by jsmeix** [comment]
Only an addendum FYI regarding https://github.com/rear/rear/issues/939#issuecomment-234862890

To get arbitrary packages fixed in the openSUSE build service
you do not need write access for the project or the package itself.

In the openSUSE build service you can branch any package
to get your own copy and do your changes on your copy
and then file a so called "submitrequest", cf.
https://en.opensuse.org/openSUSE:Package_maintenance

In short:
The openSUSE build service works like GitHub:
Changing his own copy plus "submitrequest"/"pull request".
